### PR TITLE
Ra 1061 

### PR DESCRIPTION
**Pull request by AdamGrzybkowski**
Do you know how we could pass few arguments that would e.g. fill the description in class-edit.html page ?

### CODE REVIEW

**Comment by rkorytkowski** [comment]
Not sure what you mean by arguments? Do you simply want to set fields to initial value? To achieve that you need to request in a controller a REST resource and then expose it to the view through a variable in a model. It's basically the same way as you exposed $scope.classes.

File: app/js/app.js [hunk 1]
```diff
@@ -8,6 +8,13 @@ conceptDictionary.config(['$routeProvider',
                           templateUrl: 'partials/class-list.html',
                           controller: 'ClassesListCtrl'
                         }).
+                        when('/class-list/addClass', {
+                        	templateUrl: 'partials/class-add.html'                      		
+                        }).
+                        when('/class-list/:className*', {
```
Comment (pgutkowski) [inline app/js/app.js:14]: should be "`:className`" instead of "`:className*`". Now directs straight to edit view.

File: app/js/app.js [hunk 2]
```diff
@@ -8,6 +8,13 @@ conceptDictionary.config(['$routeProvider',
                           templateUrl: 'partials/class-list.html',
                           controller: 'ClassesListCtrl'
                         }).
+                        when('/class-list/addClass', {
```
Comment (pgutkowski) [inline app/js/app.js:11]: inconsistent naming, should be `add-class`

File: app/js/app.js (hunk 1 above)
Comment (AdamGrzybkowski) [inline app/js/app.js:14]: ":className" doesn't work for names that contains slashes

**Pull request closed without merging**